src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -69,9 +69,10 @@ public function load($relations)
      * @param  array|string  $relations
      * @param  string  $column
      * @param  string  $function
+     * @param  bool  $preserveRelationState
      * @return $this
      */
-    public function loadAggregate($relations, $column, $function = null)
+    public function loadAggregate($relations, $column, $function = null, $preserveRelationState = false)
```

Comment:
This is a breaking change and cannot be done on a patch release. Please re-send this without this change. Thanks